action.yml

```diff
@@ -41,3 +41,6 @@ inputs:
   flyctl_version:
     description: Specify the version of the flyctl command-line tool to use
     required: false
+  build_secrets:
+    description: Secrets to be set at build time. Values are read as `name=value`
```

Comment:
I would just maybe add a bit about how to package it if there are multiple... Is it space-separated like `NAME_ONE=value_one NAME_TWO=value_two` ?

Reply:
@grossvogel It's just like the "secrets:" one above it. It is space separated, but also can be line-separated. It comes through as space-separated either way. So .toml for our review apps looks like:

```
        with:
          name: ${{ github.event.repository.name }}-pr-${{ github.event.number }}
          postgres: ${{ github.event.repository.name }}-pr-${{ github.event.number }}-db
          dockerfile: "Dockerfile"
          wait: "true"
          memory: 512
          secrets:
            APP_DOMAIN=${{ github.event.repository.name }}-pr-${{ github.event.number }}.fly.dev
            AMPLITUDE_API_KEY=${{ secrets.AMPLITUDE_API_KEY }}
            AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
            AWS_S3_BUCKET=${{ secrets.AWS_S3_BUCKET }}
```

How would you state that? "Space or line separated"?

Reply:
hah, I don't know it just wasn't obvious.... I guess it's OK as-is

Reply:
I did "Multiple values should be delineated by spaces or on separate lines."